bower.json

```diff
@@ -15,9 +15,10 @@
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
-    "d3": "^3.5.16",
-    "vega": "^2.5.0",
-    "vega-lite": "^1.0.7"
+    "d3": "^4.4.4",
```

Comment:
Let's only require vega and vega-lite. d3 and vega-schema-parser should be compiled into our lib.